### PR TITLE
Email form type changed to email

### PR DESCRIPTION
**Pull request by satyammittal**
#35 
fixed-"Input type of login email field should be email instead of text"

### CODE REVIEW

File: mod_auth/forms.py [hunk 1]
```diff
@@ -42,13 +42,13 @@ class LoginForm(Form):
     email = StringField('Email', [DataRequired(
         message='Email is not filled in.')])
     password = PasswordField('Password', [
-        DataRequired(message='Password cannot be empty.')])
+        DataRequired(message='Password cannot be empty.'), Email("entered value is not a valid email address")])
```
Comment (saurabhshri) [inline mod_auth/forms.py]: You probably meant to add it on #L43.
Also, please change it from `entered` to `Entered` while you are at it.

File: mod_auth/forms.py [hunk 2]
```diff
@@ -42,13 +42,13 @@ class LoginForm(Form):
...
     submit = SubmitField('Login')
 
 
 class SignupForm(Form):
     email = StringField('Email', [DataRequired(
-        message='Email is not filled in.')])
+        message='Email is not filled in.'), Email("entered value is not a valid email address")])
```
Comment (saurabhshri) [inline mod_auth/forms.py]: Same change as above.

File: mod_auth/forms.py (hunk 1 above)
Comment (satyammittal) [inline mod_auth/forms.py]: oh yes !!
Thanks

**Comment by canihavesomecoffee** [comment]
Fixes are just adding extra validation (which isn't bad on itself), but do not change the type to email, as requested in the issue.

**Comment by satyammittal** [comment]
then should i only have to validate for login?

**Comment by saurabhshri** [comment]
@satyammittal I think you misunderstood the requirement.  Currently, the input type in the generated  HTML form is of type _text_. You simply have to change it to _email_ wherever required. 


email


It has already been done in the same file you're working. Just look at it. It's just a matter of changing from `StringField` to `EmailField`.

**Pull request merged**